src/arch/Sound/RageSoundDriver_Generic_Software.cpp

```diff
@@ -7,9 +7,14 @@
 #include "RageSoundMixBuffer.h"
 #include "RageSoundReader.h"
 
+#include <cinttypes>
 #include <cmath>
 #include <cstdint>
 
+#if defined(_MSC_VER) && !defined(PRIu64)
+    #define PRIu64 "I64u"
+#endif
+
```

Comment:
I don't think this little chunk is needed. In #366 I just have `include <inttypes.h>` here, though I tested and `<cinttypes>` works just as well.

https://github.com/itgmania/itgmania/blob/658a4236cbcbc4def164a2ca2e1e7ae0fb59f52c/src/arch/Sound/RageSoundDriver_Generic_Software.cpp#L12-L16

Reply:
Pipeline failed for the last commit before I added this: https://github.com/florczakraf/itgmania/actions/runs/10199057997/job/28215288919. I did some reading about it and it seems that it's related to MS being behind in implementing C standard in their compiler. This is needed, but perhaps we could define it in some higher level header (and definitely in a separate MR -- I'll start extracting stuff from here some day)

Reply:
> Pipeline failed for the last commit before I added this: https://github.com/florczakraf/itgmania/actions/runs/10199057997/job/28215288919. I did some reading about it and it seems that it's related to MS being behind in implementing C standard in their compiler. This is needed, but perhaps we could define it in some higher level header (and definitely in a separate MR -- I'll start extracting stuff from here some day)

Weird. I can look into this, because as you probably already know (but other readers probably don't) I'm using VS 2022 with the latest MSVC.